joint_velocity_example_controller:
    type: fliqc_controller_ros/JointVelocityExampleController
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7

lcqpow:
    complementarityTolerance: 2.221E-13
    stationarityTolerance: 2.221E-10
    initialPenaltyParameter: 0.01
    penaltyUpdateFactor: 2.0

fliqc_controller_core:
    buffer_history: false
    enable_lambda_constraint_in_L: false
    enable_lambda_constraint_in_x: true
    lambda_max: 1.7976931348623157E+308
    enable_esc_vel_constraint: false
    esc_vel_max: 1.7976931348623157E+308
    dt: 0.02
    eps: 0.01
    active_threshold: 0.03
    q_dot_max: [2.1750, 2.1750, 2.1750, 2.1750, 2.6100, 2.6100, 2.6100]   # [rad/s]

fliqc_joint_velocity_no_env_node:
    type: fliqc_controller_ros/FLIQCJointVelocityNoEnvNode
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7

fliqc_joint_velocity_no_env_node_debug:
    type: fliqc_controller_ros/FLIQCJointVelocityNoEnvNodeDebug
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7

fliqc_joint_velocity_standard:
    type: fliqc_controller_ros/FLIQCJointVelocityStandard
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7

fliqc_joint_velocity_standard_debug:
    type: fliqc_controller_ros/FLIQCJointVelocityStandardDebug
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7
